html_text/html_text.py

```diff
@@ -96,7 +185,22 @@ def extract_text(html, guess_punct_space=True):
     for punctuation. This has a slight (around 10%) performance overhead
     and is just a heuristic.
 
+    When guess_page_layout is True (default is False), a newline is added
+    before and after NEWLINE_TAGS and two newlines are added before and after
```

Comment:
I think it is more precise to say `newline_tags` instead of `NEWLINE_TAGS` - "a newline is added before and after `newline_tags`", and remove a note below ("NEWLINE_TAGS and DOUBLE_NEWLINE_TAGS can be customized.") - users shouldn't be changing NEWLINE_TAGS, they should be passing `newline_tags` arguments, e.g.

```py
html_text.extract_text(html, guess_page_layout=True, newline_tags=html_text.NEWLINE_TAGS | {'div'})
```

^^ maybe we should even provide this example somewhere, adding `div` may be a common thing

Reply:
I will add an example in the readme but div is already included in NEWLINE_TAGS. I will use a different tag for clarity :)

Reply:
Or do you want me to add a test?

Reply:
Hm, that's not a bad idea - let's to do both.